source/layout.erb

```diff
@@ -52,6 +52,11 @@
       <div id="content" class="main<% if sidebar %> has-sidebar<% end %>" role="main">
         <%= yield %>
       </div>
+      <div class="cta-contribute">
+        <a title="Use GitHub to suggest an edit to this page" href="https://www.github.com/emberjs/website">
```

Comment:
Would it be a good idea to have this open in a new tab with `target= _blank`?

Reply:
I like this idea. Want me to update the pr to include `target='_blank'`  @kennethlarsen?

Reply:
If you do decide you want to go this route, please consider a new PR, ensuring that the solution you implement follows WCAG Guideline 3.2 (Predictable) and perhaps implements technique [G201: Giving users advance warning when opening a new window](https://www.w3.org/TR/WCAG20-TECHS/G201.html). Thank you! :+1: